bftengine/src/bftengine/PersistentStorageImp.cpp

```diff
@@ -615,7 +615,6 @@ PersistentStorageImp::getAndAllocateDescriptorOfLastExitFromView() {
 
 DescriptorOfLastNewView
 PersistentStorageImp::getAndAllocateDescriptorOfLastNewView() {
-  Assert(getIsAllowed());
```

Comment:
Haha, I could have perfomed this fix :) I guess it's like the repairman charging a $1 for the fix and $99 for knowing where to kick the appliance.

Reply:
Exactly, this knowledge worth its price :-)